### ISSUE TITLE
@stacks/auth not usable in app

### ISSUE THREAD

**Issue by friedger** [stx-labs/stacks.js#852]
**Describe the bug**
Importing @stacks/auth 1.0.0-beta.9 results in the error using a simple react app
```
Failed to compile.

./node_modules/@stacks/connect/dist/connect.esm.js
Module not found: Can't resolve 'blockstack' in '/home/friedger/_repos/github/friedger/speed-spend/node_modules/@stacks/connect/dist'
```
**What version of stacks.js are you using?**
1.0.0-beta.9
**Is the bug present in the Blockstack CLI, Gaia hub, Connect or Blockstack Browser?**
auth

**To Reproduce**
1. git checkout speed spend
2. update dependencies to stacks/auth
3. `yarn start`

**Expected behavior**
Just resolve the dependencies

**Additional context**
@stacks/auth (exports useConnect) uses @stacks/connect (exports useConenct, imports UserSession) uses @stacks/auth (exports UserSession)

**Comment by friedger** [comment]
Adding `blockstack` as a dependency helps as a workaround.

**Comment by alexkeating** [comment]
I think this was resolved with https://github.com/blockstack/stacks.js/pull/875

**Comment by pradel** [comment]
I am having the same issue with v1.0.0-beta.20

**Comment by pradel** [comment]
Same error with v1.0.0-beta.20 I think it was not resolved.

**Comment by markmhendrickson** [comment]
@pradel @friedger Is this still a problem if you import `stacks/connect` instead of `stacks/auth`, and does the former work for your purposes?

**Comment by pradel** [comment]
@markmhx I think this (stale) pr needs to be merged first to get rid of this issue https://github.com/blockstack/ux/pull/686

**Comment by markmhendrickson** [comment]
Sounds good. This is now blocked by the new PR https://github.com/blockstack/ux/pull/743

I'm assigning this issue to @hstove so he follows up here after that PR is merged.

**Comment by markmhendrickson** [comment]
This will go live with https://github.com/blockstack/ux/pull/761
